### ISSUE TITLE
Cloud flare issue

### ISSUE THREAD

**Issue by FleetAdmiralJakob** [pastelsky/tsdocs#32]
![Screenshot_20231227_162308_Edge.jpg](https://github.com/pastelsky/tsdocs/assets/77451351/d0b03e83-3316-4fe2-a53b-9ac19ac57417)

**Comment by nickmccurdy** [comment]
I've also been getting 502 Bad gateway intermittently for my package [`react-unpin`](https://npm.im/react-unpin): https://tsdocs.dev/docs/react-unpin

**Comment by FleetAdmiralJakob** [comment]
@nickmccurdy ![Screenshot_20231229_094048_Edge.jpg](https://github.com/pastelsky/tsdocs/assets/77451351/6e7244c6-6aad-4e87-861d-025feb76fbdd)

**Comment by nickmccurdy** [comment]
I know, the issue is intermittent.

**Comment by drewbitt** [comment]
The issue is known https://github.com/pastelsky/tsdocs/issues/28#issuecomment-1890998054 but yes is also affecting me frequently. Less than half the time recently I can get a package to show up.

**Comment by difosfor** [comment]
Yeah, poor reliability and performance (e.g: when package is not cached and has to be processed again) are the main issues keeping me nervous about using this service. I really want to use this, but I'm thinking more and more of instead perhaps running/rolling my own. Perhaps I could just setup my own server running the same code, but restricted to only support packages with our prefix 🤔
